vc-verifier/kotlin/vcverifier/src/main/java/io/mosip/vercred/vcverifier/credentialverifier/VerifiableCredential.kt

```diff
@@ -0,0 +1,8 @@
+package io.mosip.vercred.vcverifier.credentialverifier
+
+import io.mosip.vercred.vcverifier.data.VerificationResult
+
+interface VerifiableCredential {
+    fun validate(credential: String): VerificationResult
```

Comment:
validate should return only Error message. CredentialVerifier should construct the verification result. we are anyway updating verificationStatus to be same as of signature verification status.

Reply:
Updated to return ErrorMessage